lib/oracledb.js

```diff
@@ -22,13 +22,33 @@ var oracledbInst;
 var Lob = require('./lob.js').Lob;
 var pool = require('./pool.js');
 var connection = require('./connection.js');
+var path;
+var existsSync;
 
 try {
   oracledbCLib =  require('../build/Release/oracledb');
 } catch (err) {
   if (err.code === 'MODULE_NOT_FOUND') {
     oracledbCLib = require('../build/Debug/oracledb');
   } else {
+    
+    path = require('path');
+    existsSync = require('fs').existsSync || require('path').existsSync;
+    
+    process.env.PATH.split(path.delimiter).forEach(function(dir) {
+      if(existsSync(path.join(dir, 'oci.dll'))) {
+		  console.log();
+		  console.log('**** Found Oracle Instant Client in PATH in the following direcory:' + dir);
```

Comment:
This doesn't look right:
```javascript
    path = require('path');
    existsSync = require('fs').existsSync || require('path').existsSync;
```

I can't find `existsSync` in `path`. Also, existsSync is deprecated:
https://nodejs.org/api/fs.html#fs_fs_existssync_path

This the main answer here offers an alternative:
http://stackoverflow.com/questions/4482686/check-synchronously-if-file-directory-exists-in-node-js